docs/reference/statistics.md

```diff
@@ -0,0 +1,204 @@
+
+
+```python
+{
+    "start_date": None,
+    "end_date": None,
+    "recid": "fq14q-7ja92",
+    "parent_recid": "n5qej-kaz30",
+    "views": 13.0,
+    "unique_views": 6.0
+}
+```
+
+
+### Final record usage statistics
+
+The final usage statistics for a record includes the record views and file downloads
+for both the selected record version as well as across all of its versions.
+InvenioRDM turns them into the following shape:
+```python
+{
+    "this_version": {
+        "views": 10,
+        "unique_views": 6,
+        "downloads": 7,
+        "unique_downloads": 7,
+        "data_volume": 123.456,
+    },
+    "all_versions": {
+        "views": 30,
+        "unique_views": 16,
+        "downloads": 23,
+        "unique_downloads": 21,
+        "data_volume": 345.678,
+    }
+}
+```
+
+
+#### Putting the stats into the records
+
+Every record has the usage statistics available via a transient `stats` property that's
+lazy-loaded only when it is accessed.
+
+For consistency between the search results and the landing pages (and a bit of caching),
+the primary source for the collected record statistics is the records search index.
+As a fallback, the statistics are fetched directly via several queries from the
+aggregations search indices.
+
+!!! info "Outdated statistics?"
+    A special search dumper extension for records will take care of updating the
+    statistics before indexing the record in the search engine.
+    The upshot here is that when the statistics seem to be outdated, you should
+    try to reindex the record.
+
+
+## REST API endpoint
+
+`Invenio-Stats` provides a [REST API endpoint](./rest_api_statistics.md) for querying the statistics.  
+The required permissions to access this endpoint are determined by the `query_stats` entry
+in the permissions policy.
+
+!!! info "Disabled per default"
+    Per default, access to this specific API endpoint is disabled to prevent
+    attackers from overloading the system with too many or heavy queries.
+    When enabling access to the system, it should be limited to certain groups of users
+    (e.g. authenticated users, administrators, etc.) and/or rate-limiting should be
+    put in place.
+
+    However, the system will still include a record's usage statistics in the
+    web interface as well as the API endpoints for records.
+
+
+### Default queries
+
+The following queries are available per default:
+
+| Query name                     | Parameters                |
+| ------------------------------ | ------------------------- |
+| `record-view`                  | `recid` (required)        |
+| `record-view-all-versions`     | `parent_recid` (required) |
+| `record-download`              | `recid` (required)        |
+| `record-download-all-versions` | `parent_recid` (required) |
+
+On top of that, all queries accept an optional `start_date` and `end_date` parameter to
+limit the time span for which to report the statistics.
+
+
+## Additional information
```

Comment:
I have a feeling that these points here would be a better fit for the release notes, as they describe the defaults for v12.